LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright 2021 namuskaya, Daniel Doubrovkine, Nikita Bulai
```

Comment:
Let's say `Copyright Mustermann-Grape Contributors`, no year needed either.

Reply:
I vaguely remember reading somewhere that a year is required. But if included, it should probably have been the year of first publication, not the current one..

I do not know if something like "Mustermann-Grape Contributors" is valid either, as it is no legal entity.
But as you are the maintainer, I am fine with it if you are :smiley:.

Reply:
IANAL, but on opensearch.org that I am working on rn we consulted an attorney who recommended removing the year and giving copyright to contributors in this way. So I've started adopting it everywhere else. :shrug: